spec/unit/action/deploy/module_spec.rb

```diff
@@ -126,9 +119,8 @@
         expect(subject.settings[:overrides][:environments][:generate_types]).to eq(true)
       end
 
-      it 'only calls puppet generate types on environments with specified module' do
+      it 'only calls puppet generate types on environments where the specified module was updated' do
         subject.call
-        expect(@modules.length).to be(2)
       end
```

Comment:
I'm not sure what the purpose of this test is, now that there are no `expect`s.. Should this just get squashed into a single test for 'with generate-types enabled', rather than putting the meat of the test into a `before` block?